app/src/main/java/com/mstudio/superheroarch/presentation/detail/CharacterDetailViewModel.kt

```diff
@@ -4,27 +4,29 @@ import com.mstudio.superheroarch.presentation.model.CharacterData
 import com.mstudio.superheroarch.presentation.network.NetworkManager
 import com.mstudio.superheroarch.usecase.CharacterAndEpisodeData
 import com.mstudio.superheroarch.usecase.GetCharacterAndEpisodeUseCase
+import kotlinx.coroutines.CoroutineDispatcher
 import kotlinx.coroutines.CoroutineScope
 import kotlinx.coroutines.Dispatchers
 import kotlinx.coroutines.launch
 import kotlinx.coroutines.withContext
 
 class CharacterDetailViewModel(
-    private val view: CharacterDetailViewTranslator
+    private val view: CharacterDetailViewTranslator,
+    private val useCase: GetCharacterAndEpisodeUseCase = GetCharacterAndEpisodeUseCase(),
```

Comment:
makes no sense to give default values to avoid creating this in the view. That's a way to identify that you are not doing the things properly.